vp/devel/python_peripherals/DLA.py

```diff
@@ -318,7 +295,7 @@ def cast_long_to_signed_byte(value):
     Returns:
     byte -- Int Signed value in range -128..127
     """
-    assert(0 <= value <= 255)
+    assert(0 <= value <= 255), "Assert failed! Value doesn't fit byte"
```

Comment:
Might be better expressed as `assert(0 <= value <= 0xff)`

Reply:
Fixed